pytest/tests/loadtest/locust/common/ft.py

```diff
@@ -166,15 +166,20 @@ def on_locust_init(environment, **kwargs):
     num_ft_contracts = environment.parsed_options.num_ft_contracts
     funding_account = NearUser.funding_account
     parent_id = funding_account.key.account_id
+    run_id = environment.parsed_options.run_id
 
     funding_account.refresh_nonce(node.node)
 
     environment.ft_contracts = []
     # TODO: Create accounts in parallel
     for i in range(num_ft_contracts):
-        account_id = environment.account_generator.random_account_id(
-            parent_id, '_ft')
-        contract_key = key.Key.from_random(account_id)
+        if environment.parsed_options.fixed_contract_names:
+            account_id = f"ft{run_id}_{i}.{parent_id}"
```

Comment:
the `run_id` is an user-supplied value, yes? I mildly wonder if it wouldn't be more future proof to allow specifying some sort of seed rather than grabbing some random user-specified value... at the very least this is still not reproducible between different runs as far as I understand, which possibly makes the results difficult to compare potentially.

Reply:
I cannot find any documentation about this `run_id`. Do I assume correctly that in all practical runs, this will actually parse as `ftNone_{i}.{parent_id}`?

(I tried, out of fun, to ask ChatGPT to tell me about what `run_id` is, and it answered what I’d have instinctively expected, aka `run_id` being a randomly-generated ID for each run, which would break this PR because `run_id` would then not be reproducible. But I cannot find any confirmation of that from the locust documentation, unfortunately)

If it is indeed something that will parse as `ftNone_[...]`, can we just have `account_id = f"ft_{i}.{parent_id}"`? Or if you really want to have the ability to define multiple account names with a user-configurable flag, can we instead use `parsed_options.fixed_contract_name_prefix`, so that it’s self-documenting?

Reply:
`run_id` is defined in `base.py` [1], it is a string that is empty by default, so we would get `ft_{i}.{parent_id}`. This value is already used in a number of places [2] with a similar purpose of re-using the same NEAR network with multiple independent Locust workloads.

[1] https://github.com/near/nearcore/blob/66d3b134343d9f35f6e0b437ebbdbef3e4aa1de3/pytest/tests/loadtest/locust/common/base.py#L917

[2] https://github.com/search?q=repo%3Anear%2Fnearcore+path%3A%2F%5Epytest%5C%2Ftests%5C%2Floadtest%5C%2Flocust%5C%2F%2F+run_id&type=code